src/detect-engine-uint.h

```diff
@@ -24,39 +24,40 @@
 #ifndef __DETECT_ENGINE_UINT_H
 #define __DETECT_ENGINE_UINT_H
 
+#include "rust.h"
 #include "detect-engine-prefilter-common.h"
 
-typedef enum {
-    DETECT_UINT_LT = PREFILTER_U8HASH_MODE_LT,
-    DETECT_UINT_EQ = PREFILTER_U8HASH_MODE_EQ,
-    DETECT_UINT_GT = PREFILTER_U8HASH_MODE_GT,
-    DETECT_UINT_RA = PREFILTER_U8HASH_MODE_RA,
-    DETECT_UINT_LTE,
-    DETECT_UINT_GTE,
-} DetectUintMode;
-
-typedef struct DetectU32Data_ {
-    uint32_t arg1;   /**< first arg value in the signature*/
-    uint32_t arg2;   /**< second arg value in the signature, in case of range
-                          operator*/
-    DetectUintMode mode;    /**< operator used in the signature */
-} DetectU32Data;
-
-int DetectU32Match(const uint32_t parg, const DetectU32Data *du32);
-DetectU32Data *DetectU32Parse (const char *u32str);
+// These definitions are kept to minimize the diff
+// We can run a big sed commit next
+#define DETECT_UINT_GT  DetectUintModeGt
+#define DETECT_UINT_GTE DetectUintModeGte
+#define DETECT_UINT_RA  DetectUintModeRange
+#define DETECT_UINT_EQ  DetectUintModeEqual
+#define DETECT_UINT_NE  DetectUintModeNe
+#define DETECT_UINT_LT  DetectUintModeLt
+#define DETECT_UINT_LTE DetectUintModeLte
+
+typedef DetectUintData_u64 DetectU64Data;
```

Comment:
These don't match our style, should probably be `DetectUintDataU32`

Reply:
What do you mean ?
`DetectUintData_u64` is generated by cbindgen
`DetectU64Data ` is what is in the code currently

I added these typedef to minimize the diff.
But we can run sed 's/DetectU64Data/DetectUintData_u64/' and avoid these